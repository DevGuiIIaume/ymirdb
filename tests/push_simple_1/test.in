push a 1
push a 1 2
set a 1 2 3
get a
push a 0
list entries
push a -1
list entries
push a -2 -3
get a
bye
