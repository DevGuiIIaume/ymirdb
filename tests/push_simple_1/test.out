> no such key

> no such key

> ok

> [1 2 3]

> ok

> a [0 1 2 3]

> ok

> a [-1 0 1 2 3]

> ok

> [-3 -2 -1 0 1 2 3]

> bye
